API.md

```diff
@@ -148,80 +148,73 @@ Returns **cc.Condition** Ed25519 Condition (that will need to wrapped in an Outp
 
 Returns **[object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)** Ed25519 Condition (that will need to wrapped in an Output)
 
-## getBlock
+## Connection
```

Comment:
did you create these using documentation.js?

`npm install -g documentation`
`documentation build src/index.js -a public -f md > API.md  `